doc/sphinx/examples_cpp/basics/basics.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2017 Sebastian Weber, Henri Menke. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <boost/filesystem.hpp>
+#include <stdexcept>
+#include <iostream>
+
+#include "libpairinteraction/dtypes.h"
+#include "libpairinteraction/State.h"
+#include "libpairinteraction/SystemOne.h"
+#include "libpairinteraction/SystemTwo.h"
+
+int main() {
+    // Create cache directory
+    boost::filesystem::path path_cache =
+        boost::filesystem::temp_directory_path()/boost::filesystem::unique_path();
+    if (boost::filesystem::create_directory(path_cache)) {
+        std::cout << "Cache directory "
+                    << boost::filesystem::absolute(path_cache).string()
+                    << " created." << std::endl;
+    } else {
+        throw std::runtime_error("Could not create cache directory.");
```

Comment:
Does this blow up if `path_cache` exists already?

Reply:
Exactly. With my last commit, I have completely changed this part of code. Now, the directory "./cache" is created as in the Python tutorials.